.github/workflows/run-tests.yml

```diff
@@ -1,10 +1,11 @@
 name: Run Build and Tests
 
-on: push
+on: [push, pull_request]
```

Comment:
> This would mean that the CI would no longer run for branches within the repo that are not associated with any pull request. On one had that is a "regression" in functionality

i guess i'd only call it a regression if we're affected _desired_ behavior. do we actually want to tests for commits w/o a PR for a concrete reason? what's the user scenario